auth-server/README.md

```diff
@@ -395,6 +395,35 @@ On success, we get the response
 }
 ```
 
+### Resend Email verification Token
```

Comment:
Thanks a lot for taking the time to do this. Thinking about it, I don't think that we should bother about documenting our endpoints in the future since everything is accessible in graphiql. I removed most of it in #283

Reply:
ah, then removed